### PR TITLE
fixed hbar tick label overlapping

### LINKED ISSUE #426: detailed_program_spending_split issues with sliced-hbar
Screen Shot 2019-12-12 at 3 47 21 PM

### PR DESCRIPTION
**Pull request by ktw1016**
fixes #426

### CODE REVIEW

File: client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js [hunk 1]
```diff
@@ -252,6 +251,8 @@ class DetailedProgramSplit extends React.Component {
       .sortBy("total")
       .value();
 
+    const divHeight = 650 * (graph_ready_data.length / 30) * 2;
```
Comment (Stephen-ONeil) [inline client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js]: Hmm, where does the 30 and the 2 come from here. Just magic numbers that get the positioning right?

Wish nivo's bar graphs let us explicitly pass in a bar width (or height in this case).

Comment (ktw1016) [reply client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js]: Yup these are just magic numbers I tried

Comment (Stephen-ONeil) [reply client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js]: Haha, hmm. I looked at it a bit and saw some other magic numbers/formulas in that file, so this isn't making it much worse overall.

Just leave a comment about why the magic calculation's there/what it's doing and I'll approve 👍

Comment (Stephen-ONeil) [reply client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js]: Oh, my page refreshed when I submitted that comment and your commit doing it showed up

File: client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js [hunk 2]
```diff
@@ -251,6 +251,7 @@ class DetailedProgramSplit extends React.Component {
       .sortBy("total")
       .value();
 
+    // Change height of the graph region according to the number of programs
```
Comment (Stephen-ONeil) [inline client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js]: Hmm, sorry to nitpick, but this is more of the "what", I was thinking more of the "why". Maybe add a second sentence saying something like "This is required for y-axis labels to have sufficient room when many programs are present".

File: client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js (hunk 1 above)
Comment (ktw1016) [reply client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js]: okay cool! so do u think the comment I made is sufficient?

Comment (Stephen-ONeil) [reply client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js]: Seems good now!

**Pull request merged**